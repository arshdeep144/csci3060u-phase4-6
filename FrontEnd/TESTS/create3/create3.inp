login
testingusername1
logout
