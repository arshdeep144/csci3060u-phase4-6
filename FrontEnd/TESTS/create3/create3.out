Welcome to the Online Auction System! Please login to Proceed
Please enter your username: 
Username not found in database
